matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/crypto/keysbackup/KeysBackupService.kt

```diff
@@ -204,10 +202,13 @@ interface KeysBackupService {
                                      callback: MatrixCallback<ImportRoomKeysResult>)
 
     val keysBackupVersion: KeysVersionResult?
+
     val currentBackupVersion: String?
-    val isEnabled: Boolean
-    val isStucked: Boolean
-    val state: KeysBackupState
+        get() = keysBackupVersion?.version
+
+    fun isEnabled(): Boolean
+    fun isStucked(): Boolean
```

Comment:
I guess it should be `isStuck()` instead. The version with "ed" seems to not exist.

Reply:
Ah yes, will add.

Reply:
eb576808b97179969a95ff81a0b2daf7277dcb00